### ISSUE TITLE
Suggestion: pure Rust through tether

### ISSUE THREAD

**Issue by LeDominik** [huytd/kanban-app#3]
Really really like this -- but after researching the HiDPI and MSHTML engine (prefer Edge if having to deal with Windows) I stumbled upon [tether](https://github.com/quadrupleslap/tether) which seems to be a nearly-pure-Rust version to get a WebView up...

**Comment by huytd** [comment]
Interesting. Thank you so much! I will take a look at this. 

Maybe we'll have multiple branches using multiple webview libraries, so people will have many options for this.

**Comment by LeDominik** [comment]
Yeah, it's pretty interesting times. A good webview lib with some features like menus, etc... makes a quite solid foundation for any app 👍
